src/pages/iou/IOUModal.js

```diff
@@ -434,7 +434,11 @@ class IOUModal extends Component {
                                                 comment={this.state.comment}
                                                 onUpdateComment={this.updateComment}
                                                 iouType={this.props.iouType}
-                                                isGroupSplit={this.steps.length === 2}
+
+                                                // Note: We do not allow participants to be adjusted if this was opened from the global create menu,
+                                                // because if a user reaches the confirm step from that flow, they've already hand-picked the participants.
+                                                // eslint-disable-next-line react/jsx-props-no-multi-spaces
+                                                canDeselectParticipants={this.steps.length === 2}
```

Comment:
Ok got it. Seeing this comment does help. So we should do `canModifyParticipants` (assuming that this also means we won't be able to change these participants at all)?